File: account/src/main/java/com/dollee/bank/account/config/SwaggerConfig.java
package com.dollee.bank.account.config;

import io.swagger.v3.oas.annotations.OpenAPIDefinition;
import io.swagger.v3.oas.annotations.info.Info;
import io.swagger.v3.oas.models.Components;
import io.swagger.v3.oas.models.OpenAPI;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
@OpenAPIDefinition(info = @Info(title = "Dollee Bank - Account Swagger", description = "Dollee Bank - Account REST API"))
public class SwaggerConfig {

  @Bean
  public OpenAPI openAPI() {
    return new OpenAPI()
        .components(new Components())
        .info(apiInfo());

  }

  private io.swagger.v3.oas.models.info.Info apiInfo() {
    return new io.swagger.v3.oas.models.info.Info()
        .title("Dollee Bank - Account Swagger")
        .description("Dollee Bank - Account 관한 REST API")
        .version("1.0.0");
  }
}

File: account/src/main/java/com/dollee/bank/account/config/SwaggerConfig.kt
package com.dollee.market.dolleeMarket.config

import io.swagger.v3.oas.models.Components
import io.swagger.v3.oas.models.OpenAPI
import io.swagger.v3.oas.models.info.Info
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class SwaggerConfig {
    @Bean
    fun swaggerApi() : OpenAPI = OpenAPI().components(Components())
        .info(swaggerInfo())

    private fun swaggerInfo() = Info()
        .title("스웨거 테스트")
        .description("스웨거로 API를 테스트")
        .version("1.0.0")
}